src/main/java/com/MTAPizza/Sympoll/pollmanagementservice/dto/vote/delete/VoteDeleteRequest.java

```diff
@@ -0,0 +1,9 @@
+package com.MTAPizza.Sympoll.pollmanagementservice.dto.vote.delete;
+
+import java.util.UUID;
+
+public record VoteDeleteRequest(
```

Comment:
How will the browser get the vote id?
Is it something that will be returned to it after a vote and is tagged onto an checked answer?

Reply:
The browser doesn't comunicate with the vote endpoints of the poll service.